torch/csrc/jit/codegen/cuda/ops/alias.cpp

```diff
@@ -129,6 +129,25 @@ TensorView* flatten(TensorView* x, int64_t start_dim, int64_t end_dim) {
     return x;
   }
 
+  std::vector<int> trivial_reduction_dims;
```

Comment:
@zasdfgbnm please review this change too.

Reply:
I don't think this is correct. We don't want to skip just trivial reduction, but we need to do a `TensorDomain::noReductions` before any operations. See my updated commit.

Reply:
Yeah, thanks, your commit looks good.